<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelEditorClass</class>
 <widget class="QMainWindow" name="LevelEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Voiceover</string>
     </attribute>
     <widget class="QTextEdit" name="voiceover_textEdit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>270</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addTextButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>400</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>AddText</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadImgButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>450</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
     <widget class="QListWidget" name="voiceover_listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="bg_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>320</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string>No Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="local_img_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="editTextButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>400</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>EditText</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteTextButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>400</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DeleteText</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>WorldMap</string>
     </attribute>
     <widget class="QScrollBar" name="worldframe_scrollBar">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>460</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWorldMapFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>640</width>
        <height>400</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="world_map_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>640</width>
         <height>400</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="cur_frame_label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>460</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ADMUI3Lg</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>cur_frame:</string>
      </property>
     </widget>
     <widget class="QLabel" name="frame_num_label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>480</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ADMUI3Lg</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>frame num:1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cur_frame_lineEdit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>460</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newframe_pushButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>460</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>New Frame</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="worldMap_textEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>250</y>
        <width>91</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="defineTxt_pushButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>400</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>defineTxt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QPlotFrame" name="frame2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>640</width>
        <height>400</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="plotbg_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>640</width>
         <height>400</height>
        </rect>
       </property>
       <property name="text">
        <string>No Image</string>
       </property>
      </widget>
     </widget>
     <widget class="QScrollBar" name="plot_scrollBar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="scenename_lineEdit">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>scene name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cur_plotframe_lineEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>450</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="cur_plotframe_label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>450</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ADMUI3Lg</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>cur_frame:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newplotframe_pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>New Frame</string>
      </property>
     </widget>
     <widget class="QLabel" name="plotframe_num_label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>470</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ADMUI3Lg</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>frame num:1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>plot name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="plotname_lineEdit">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>80</y>
        <width>211</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Dialog Box</string>
      </property>
      <widget class="QTextEdit" name="plot_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>181</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="insertPlotDialog_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>insertDialog</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="charactername_lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>character</string>
       </property>
      </widget>
      <widget class="QLabel" name="characterImg_label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>64</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>No Image</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>leftUp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rightUp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>leftDown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rightDown</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="loadCharacterImg_pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Img</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>390</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>PlotItem</string>
      </property>
      <widget class="QPushButton" name="changeItemImg_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Img</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="plotItemlabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>48</width>
         <height>64</height>
        </rect>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="mirror_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mirror</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Xml"/>
    <addaction name="actionSave_Xml"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Xml">
   <property name="text">
    <string>Load Xml</string>
   </property>
  </action>
  <action name="actionSave_Xml">
   <property name="text">
    <string>Save Xml</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWorldMapFrame</class>
   <extends>QFrame</extends>
   <header>qworldmapframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPlotFrame</class>
   <extends>QFrame</extends>
   <header>qplotframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="leveleditor.qrc"/>
 </resources>
 <connections/>
</ui>
